appdata.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/src/org.kde.krename.appdata.xml b/src/org.kde.krename.appdata.xml
+index 49f4d31..f6e7f34 100644
+--- a/src/org.kde.krename.appdata.xml
++++ b/src/org.kde.krename.appdata.xml
+@@ -112,5 +112,10 @@
+   <project_group>KDE</project_group>
+   <provides>
+     <binary>krename</binary>
++    <launchable type="desktop-id">org.kde.krename.desktop</launchable>
++    <content_rating type="oars-1.1"/>
```

Comment:
this is in the wrong place in the file. Definitely not inside `<provide>`. That' why the build still fails.

Reply:
so i should close provide after binary tag?